apex-app-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -99,7 +99,7 @@
                </descriptors>
                <archiverConfig>
                  <defaultDirectoryMode>0755</defaultDirectoryMode>
-               </archiverConfig>                  
+               </archiverConfig>
```

Comment:
Can you avoid this and the other formatting changes. They could be made in a separate PR.